go-controller/pkg/ovn/loadbalancer/lb_cache.go

```diff
@@ -0,0 +1,376 @@
+package loadbalancer
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+var globalCache *LBCache
```

Comment:
Can this be simpliified to a sync.Map[string]*LBCache?

Reply:
Doubtful it will make any performance difference; most actions are full map scans. None of the locked sections are particularly hot, and none of them are bound on nbdb operations.